### ISSUE TITLE
Progress bar for screenreader users

### ISSUE THREAD

**Issue by matthilo96** [it-at-m/muc-patternlab-vue#603]
**Is your feature request related to a problem? Please describe.**
Currently, screenreader users cannot use the progress bar.

**Describe the solution you'd like**
Users can identify the number of steps in the process, the current step, and completed steps.
e.g. „Schritt 1 von 4: Leistung – erledigt“. / Aktueller Schritt Schritt 2 von 4: Termin
